charts/zalenium/templates/_pod-template.yaml

```
@@ -107,6 +107,8 @@ spec:
           value: {{ .Values.hub.keepOnlyFailedTests | quote }}
         - name: RETENTION_PERIOD
           value: {{ .Values.hub.retentionPeriod | quote }}
+        - name: CONTEXT_PATH
+          value: {{ .Values.ingress.contextpath | quote }}
```

Can you please use the same name `path` as the Kubernetes Ingress object. It is just more intuitive IMO.
```suggestion
           value: {{ .Values.ingress.path | quote }}
```